django-stubs/db/models/fields/json.pyi

```diff
@@ -13,7 +13,12 @@ from typing_extensions import Self
 from . import Field
 from .mixins import CheckFieldDefaultMixin
 
-class JSONField(CheckFieldDefaultMixin, Field):
+# __set__ value type
+_ST = TypeVar("_ST", contravariant=True, default=Any)
+# __get__ return type
+_GT = TypeVar("_GT", covariant=True, default=Any)
```

Comment:
I don't know how far or if we want to lean in to `PEP 696` (`default=` for `TypeVarLike`) as it's not approved yet

Refs:

- Pyright: https://github.com/microsoft/pyright/issues/4078#issuecomment-1666959148
- Mypy: https://github.com/python/mypy/issues/14851#issue-1613735295 (CC @intgr)
- Us: #1264 
- Steering council: https://github.com/python/steering-council/issues/177#issuecomment-1538925572

Not saying we should or shouldn't, just wanted to highlight for a possible discussion

Reply:
As long as it doesn't break any other typecheckers, I'm +1 for adding `default=`.

(PEP 696 in mypy would be very useful, we could drop a few ugly hacks like `_QuerySet` vs `QuerySet` vs `_QuerySetAny`)

Reply:
Ah didn't realize that it is still not approved. It seems like it does not change existing behavior since our existing tests are passing?

Reply:
I can be +1 too, lets try it out. Though we don't really have tests for most annotations, but these 2 are quite involved in the plugin and thus its tests.

No need to add any tests to the suite here I think. And in other cases we shouldn't need any tests for trivial uses of a `TypeVar`